### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 46

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#234] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152255514-2c2dca7c-36cb-44a6-b14c-c25a356379a4.png)

**Comment by i0nut02** [comment]
1. (6+ 7+ 5 scelgo 5 )
2. (6 scelgo 2) * (7+5 scelgo 2)   ----> non sono sicuro sia giusto
3. (6 + 7+ 5 scelgo 5) - (  (5 scelgo 5) + (7 scelgo 5) + (6 scelgo 5) - (7 + 5 scelgo 5 ) - (7 +6 scelgo 5)    ) 
 nel 3 ho fatto tutti i casi possibili - tutti i casi non possibili, nel calcolo non ho considerato l'intersezione 
frutta int cioccolato , ma ho sottratto crema int cioccolato (perché non ha la frutta) e crema int frutta, e il caso crema int cioccolato int frutta è all' interno dei casi possibili

**Comment by lord-marcel** [comment]
![46](https://user-images.githubusercontent.com/36630876/152653239-08de6fe3-60d3-4b1c-a591-12b0ee3f7f75.png)

**Comment by i0nut02** [comment]
la questione nella terza parte non è corretta per due motivi ma principalmente quando ti chiede ALMENO o fai per tipizzazione o per complemento (detto anche da Carlucci che viene un risultato diverso e sbagliato facendolo nel tuo modo).

**Comment by lord-marcel** [comment]
Nelle dispense di Carlucci non sembra esserci un paragrafo dedicato a questo caso, oppure non l'ho trovato. Ma qua c'è una immagine presa dal libro da lui consigliato in cui viene discusso questo tipo di problema, e quella di un'altra fonte che tratta lo stesso. L'uso del PIE è molto utile per problemi in cui il vincolo è un tetto massimo, tipo 'nessun gusto più di due volte'. In casi come questo va bene la mia soluzione.
![IMG_20220206_101126](https://user-images.githubusercontent.com/36630876/152674736-f0b417d6-073c-4e1a-bc18-72569aa0ad2e.jpg)
![IMG_20220206_102131](https://user-images.githubusercontent.com/36630876/152674739-53cf6fe9-2ffe-4a67-a012-8bc71b7706de.jpg)

**Comment by Rurik-D** [comment]
La mia soluzione per il 3° è la seguente:
<center>
Effettuo un passaggio al complemento
</center>

$$
senzaFrutta = \binom{7 + 5}{5} = \binom{12}{5}
$$

$$
senzaCioccolato = \binom{7 + 6}{5} = \binom{13}{5}
$$

<center>
Calcolo l'intersezione tra i due valori, cioè i gelati senza frutta e senza cioccolato (i gelati di sole creme)
</center>

$$
senzaFrut/Cioc = \binom{7}{5}
$$

<center>
Utilizzo il PIE
<center>

$$
gelati = \binom{18}{5} - \binom{12}{5} - \binom{13}{5} + \binom{7}{5}
$$

**Comment by xGancil** [comment]
Per la 3° soluzione l'ho pensata in un altro modo:
(6 scelgo 1) * (5 scelgo 1) * (16 scelgo 3), in questo modo ho sicuramente i 2 gusti scelti per poi completare con i gusti restanti.

**Comment by CasuFrost** [comment]
Mi trovo d'accordo con la soluzione di Rurik-D
